modules/tinymce/README.md

```diff
@@ -82,6 +82,6 @@ Docs are hosted on Github in the [tinymce-docs](https://github.com/tinymce/tinym
 
 [How to contribute](https://github.com/tinymce/tinymce-docs/blob/main/CONTRIBUTING.md) to the docs, including a style guide, can be found on the GitHub repo.
 
-[Documentation](https://www.tiny.cloud/docs/tinymce/7/)
+[Documentation](https://www.tiny.cloud/docs/tinymce/8/)
```

Comment:
Should we update to point to `/latest` since its redirected from `8x`?
```suggestion
[Documentation](https://www.tiny.cloud/docs/tinymce/latest/)
```

Reply:
> We don’t want documentation links be latest because someone might look at an old version and need links to that version’s documentation